src/magic_folder/cli.py

```diff
@@ -993,6 +1052,7 @@ class MagicFolderCommand(BaseOptions):
     stderr = sys.stderr
 
     subCommands = [
+        ["init", None, InitializeOptions, "Initialize a Magic Folder daemon."],
         ["create", None, CreateOptions, "Create a Magic Folder."],
```

Comment:
I wonder if "init" vs "create" will be memorable for folks (whoever the folks that are directly interacting with this CLI are...).  I don't have any suggestions for different words to use, though.

Reply:
My secret plan is to re-think all the CLI commands .. but not necessarily all at once.

In some ways, maybe in makes sense for `init` to be `magic-folder create` and then there's a  `magic-folder add` or similar to create a new magic-folder inside the ... "magic folder daemon" (I don't have a better term yet) ... or we could just re-name `create` to `add` (now .. or later) and leave `init`. In any case, we probably need a common term for "magic-folder the whole daemon process" versus "magic-folder, the thing you can add many of to the daemon process".